gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/testing/LocalDatastoreHelper.java

```diff
@@ -65,28 +63,27 @@
 /**
  * Utility to start and stop local Google Cloud Datastore process.
  */
-public class LocalGcdHelper {
-  private static final Logger log = Logger.getLogger(LocalGcdHelper.class.getName());
-
-  private final String projectId;
-  private Path gcdPath;
-  private Process startProcess;
-  private ProcessStreamReader processReader;
-  private ProcessErrorStreamReader processErrorReader;
-  private final int port;
-
-  public static final String DEFAULT_PROJECT_ID = "projectid1";
-  public static final int DEFAULT_PORT = 8080;
+public class LocalDatastoreHelper {
+  private static final Logger log = Logger.getLogger(LocalDatastoreHelper.class.getName());
   private static final String GCD_VERSION = "v1beta3";
-  private static final String GCD_BUILD = "1.0.0";
+  private static final String GCD_BUILD = "1.0.1";
+  private static final double DEFAULT_CONSISTENCY = 0.9;
   private static final String GCD_BASENAME = "gcd-" + GCD_VERSION + "-" + GCD_BUILD;
   private static final String GCD_FILENAME = GCD_BASENAME + ".zip";
-  private static final String MD5_CHECKSUM = "72156cc993835c57f72789519b85249b";
+  private static final String MD5_CHECKSUM = "df876ba8f054d69acff30ec9540ec386";
   private static final URL GCD_URL;
   private static final String GCLOUD = "gcloud";
   private static final Path INSTALLED_GCD_PATH;
   private static final String GCD_VERSION_PREFIX = "gcd-emulator ";
-  private static final double DEFAULT_CONSISTENCY = 0.9;
+  private static final String PROJECT_ID_PREFIX = "test-id-";
```

Comment:
I can change this to "test-project-" (dashes instead of underscores because the client checks for a valid project ID, and underscores aren't allowed).